### PR TITLE
Release v0.4.9

### PR DESCRIPTION
**Pull request by yulianedyalkova**
## Checklist

- [ ] Consider SIG UX feedback.
- [x] Update changelog in CHANGELOG.md.
- [ ] If adding a new type, ensure that it is added to the [CRD unit tests](https://github.com/giantswarm/apiextensions/blob/d78aba91d578d56ef49f58e336035d35edc4e1b0/pkg/crd/crd_test.go#L9).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 
+## [0.4.8] 2020-07-07
```
Comment (MarcelMue) [inline CHANGELOG.md:13]:
```suggestion
## [0.4.9] 2020-07-07
```

**Comment by MarcelMue** [comment]
Should probably also include a line for this change:
https://github.com/giantswarm/apiextensions/pull/503/files

**Comment by yulianedyalkova** [comment]
> Should probably also include a line for this change:
> https://github.com/giantswarm/apiextensions/pull/503/files

That change got merged into `add-ref-releases` not `master`.

**Comment by MarcelMue** [comment]
> > Should probably also include a line for this change:
> > https://github.com/giantswarm/apiextensions/pull/503/files
> 
> That change got merged into `add-ref-releases` not `master`.

Oh you're right :facepalm:

**Comment by kopiczko** [comment]
https://intranet.giantswarm.io/docs/dev-and-releng/how-to-release-a-project/ please <3

**Comment by yulianedyalkova** [comment]
@kopiczko sorry, will use that process next time! Anything that I broke and need to fix because of the manual release?

**Comment by kopiczko** [comment]
No worries. For Go library we only need the tag.

**Pull request merged**